.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
The live server file is also getting committed!

Reply:
will it create any problem? how is it happening?

Reply:
I dont think it is creating any impact but we can gitignore these files as they are creating a separate folder and will not look good thinking as overall project structure. I faced the similar issue then @hamza pointed this out..